cllc-public-app/ClientApp/src/app/components/sep/sep-application/event/event.component.scss

```diff
@@ -1,3 +1,7 @@
+.wrapp {
+    max-width: 900px;
+}
+
```

Comment:
I couldn't find any reference to this class in the HTML template. Also, maybe rename to **wrap**? thanks!